src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -10,9 +10,9 @@
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- * 
+ *
  *      http://www.apache.org/licenses/LICENSE-2.0
- * 
+ *
```

Comment:
please avoid unnecessary whitespace changes; this is likely to break the license-plugin check in the build